docs/rancher/csi-driver.md

```diff
@@ -99,3 +99,101 @@ The Harvester CSI driver should now be deployed successfully.
 - Install `Harvester CSI Driver` from the Rancher marketplace.
 
   ![](/img/v1.2/rancher/install-harvester-csi-driver-in-k3s.png)
+
+### Various StorageClass Supported
+
+Starting with Harvester CSI driver v0.1.15, you can create a PersistentVolumeClaim (PVC) based on a different StorageClass.
+
+The following RKE2 versions are supported for Harvester CSI driver v0.1.15 and later:
+- v1.23.16+rke2r1 and later
+- v1.24.10+rke2r1 and later
+- v1.25.6+rke2r1 and later
+- v1.26.1+rke2r1 and later
+- v1.27.1+rke2r1 and later
+
+#### Prerequisites
+
+Please complete these perquisites on the Harvester Cluster. The Harvester CSI driver needs **RBAC** to expose error messages. This is important for displaying an error message when creating a PVC with a StorageClass that does not exist, as shown in the following figure.
+
+![](/img/v1.2/rancher/error_event_csi_driver.png)
+
+
+Perform the following steps to setup **RBAC** to enable seeing error messages.
+
+1. Create a new `clusterrole` named `harvesterhci.io:csi-driver` with the following manifest.
+```
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/component: apiserver
+    app.kubernetes.io/name: harvester
+    app.kubernetes.io/part-of: harvester
+  name: harvesterhci.io:csi-driver
+rules:
+- apiGroups:
+  - storage.k8s.io
+  resources:
+  - storageclasses
+  verbs:
+  - get
+  - list
+  - watch
+```
+
+1. Then create the `clusterrolebinding` to associate with  the new `clusterrole` with the following manifest.
```

Comment:
The number list on the preview page needs to be corrected. It should be `2.` but shown as `1.`.